Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -782,19 +782,33 @@ def image_from_vtk_image(vtk_image: "vtk.vtkImageData") -> "itkt.ImageBase":
     l_origin = [0.0] * dim
     l_origin[:dim] = vtk_image.GetOrigin()[:dim]
     l_image.SetOrigin(l_origin)
+<<<<<<< HEAD
```

Comment:
Unresolved conflicts?

Reply:
yeah, clearly I've botched it up.  trying to fix it. :)